test/services/websites/create_test.rb

```diff
@@ -11,7 +11,7 @@ class CreateTest < ActiveSupport::TestCase
       Websites::Create.call(params, user)
 
       user.reload
-      assert_equal 1, user.site_number, 'site number'
+      assert_equal 1, user.websites.count, 'websites_count'
```

Comment:
I meant renaming `site_number` to `websites_count` since that is more accurately describing what it contains, not `user.websites.count`.